2014-15/C10X Episode file tests.sps

```diff
@@ -38,6 +37,44 @@ If cij_pattype = "Non-Elective" CIJ_non_elective = 1.
 If cij_pattype = "Maternity" CIJ_maternity = 1.
 If cij_pattype = "Other" CIJ_other = 1.
 
+*Flag to count how many episodes in each HB by treatment code. 
+If hbrescode = 'S08000015' NHS_Ayrshire_and_Arran = 1.
+If hbrescode = 'S08000016' NHS_Borders = 1. 
+If hbrescode = 'S08000017' NHS_Dumfries_and_Galloway = 1.
+If hbrescode = 'S08000019' NHS_Forth_Valley = 1. 
+If hbrescode = 'S08000020' NHS_Grampian = 1. 
+If any(hbrescode, 'S08000021', 'S08000031') NHS_Greater_Glasgow_and_Clyde = 1.
+If hbrescode = 'S08000022' NHS_Highland = 1.
+If any(hbrescode, 'S08000023', 'S08000032') NHS_Lanarkshire =1. 
+If hbrescode = 'S08000024' NHS_Lothian = 1. 
+If hbrescode = 'S08000025' NHS_Orkney = 1. 
+If hbrescode = 'S08000026' NHS_Shetland = 1. 
+If hbrescode = 'S08000028' NHS_Western_Isles = 1. 
+If any(hbrescode, 'S08000018', 'S08000029') NHS_Fife = 1. 
+If any(hbrescode, 'S08000027', 'S08000030') NHS_Tayside = 1. 
+
+*Change missing HB values to 0. 
```

Comment:
I don't think this is needed as SPSS will treat missing as zero in the sum anyway.